dmr/src/main/java/org/jboss/hal/dmr/Operation.java

```diff
@@ -189,6 +189,9 @@ public Builder(ResourceAddress address, String name) {
             this.parameter = new ModelNode();
             this.header = new ModelNode();
             this.roles = new HashSet<>();
+            if (name.equals(READ_RESOURCE_OPERATION)) {
+            parameter.get(RESOLVE_EXPRESSION).set(true);
```

Comment:
This will add the flag to *all* read-resource operations. I'd rather add it only when necessary as suggested at https://github.com/hal/console/pull/403#issuecomment-649362682

Reply:
@hpehl I have done the changes,
can you review it?